docs/maatregelen/richt_gebruikersbeheer_in.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
@noutvandeijck : Flink wat gesleuteld aan deze maatregelen maar heel blij om deze te zien! Goed werk en ontzettend belangrijk. In de praktijk vraagt dit om veel tijd om het netjes in te richten. Mooi dat het nu wordt toegevoegd aan het kader.